drivers/flash/Kconfig.silabs

```diff
@@ -19,4 +19,8 @@ config SOC_FLASH_SILABS_S2_DMA_WRITE
 	bool "Use DMA for flash write operations"
 	select DMA
 
+config SOC_FLASH_SILABS_S2_DMA_READ
+	bool "Use DMA for flash read operations"
+	select DMA
```

Comment:
Does it make sense to provide a such level of granularity? Having `SOC_FLASH_SILABS_S2_DMA_READ` and `SOC_FLASH_SILABS_S2_DMA_WRITE` clutter the code a lot. `SOC_FLASH_SILABS_S2_DMA` would be probably sufficient.

Reply:
Arf, I remember now. Using DMA for read operations is not really a good idea.